aeppl/mixture.py

```diff
@@ -300,6 +302,50 @@ def mixture_replace(fgraph, node):
     return [new_mixture_rv]
 
 
+@local_optimizer((Elemwise,))
```

Comment:
```suggestion
@local_optimizer((at.switch,))
```

Does that (and removing the `isinstance` below) work, as well?

Reply:
oh yes, it does! I didn't know that this was possible